cypress/integration/creatorOnboardingFlows/creatorSignup.spec.js

```diff
@@ -75,7 +75,7 @@ describe('Creator Signup Page', () => {
     cy.findByTestId('mask-icon').should('not.be.visible');
   });
 
-  it.skip("should allow sign the user in when 'Create my account' is clicked", () => {
+  it.skip("should sign the user in when 'Create my account' is clicked", () => {
```

Comment:
This change just slightly adjusts the wording of this expectation and removes the word "allow." 😄